opencti-platform/opencti-front/src/utils/String.jsx

```diff
@@ -133,3 +133,7 @@ export const splitMultilines = (str) => (str ?? '')
   .split(/\r?\n/)
   .filter((v) => !!v)
   .map((s) => s.trim());
+
+export const toggleTokenVisibility = (currentVisibility) => !currentVisibility;
+
+export const maskToken = (api_token) => '•'.repeat(api_token.length);
```

Comment:
I got error ":"TypeError: Cannot read properties of undefined (reading 'length')\n    at maskToken (http://localhost:3000/src/utils/String.jsx:136:62)\n " again while testing PR locally, please add test `if(api_token)`